stdlib/libblastrampoline_jll/Project.toml

```diff
@@ -1,13 +1,13 @@
 name = "libblastrampoline_jll"
 uuid = "8e850b90-86db-534c-a0d3-1478176c7d93"
-version = "5.8.0+1"
+version = "5.9.0+0"
 
 [deps]
 Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 Artifacts = "56f22d72-fd6d-98f1-02f0-08ddc0907c33"
 
 [compat]
-julia = "1.10"
+julia = "1.11"
```

Comment:
If you want to change this, it's 1.12, not 1.11, no?

Reply:
I was not sure because Julia 1.11 was not released yet.
But I think that `1.12` is better and we can backport it later.

Reply:
Feature freeze of 1.11 was long ago and binary dependencies are usually not backported, unless there's a compelling reason to do that (security fixes, real big fixing, etc...)

Reply:
I recall that Viral requested twice to backport a new release of LBT: https://github.com/JuliaLang/julia/pull/49281#discussion_r1160797775
I thought it was common. :sweat_smile:

Reply:
My two cents for this PR are that it changes very little and is low risk - but adds the ability to use various new symbols. Probably any breakage can be fixed very quickly. Would certainly be nice to backport to 1.11.